### PR TITLE
Add issue template

### LINKED ISSUE #1000: Issue #1000, Yay! Should we add a issue template to celebrate the occasion?
Title says it all, thoughts?

### PR DESCRIPTION
**Pull request by ecdye**
This also contains revisions to the 'openhabian-DEBUG.md' document that
help to clarify debugging procedures which go with the new issue
template addition.

Fixes #1000

Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: openHABian issue template
+about: A template issue for the openHABian repository.
+labels: new
+---
+## Issue information:
+Please be as detailed as possible to help us help you. Also, please be sure to
+check that your issue has not already been reported to the [openHAB community](https://community.openhab.org/t/13379)
```
Comment (mstormi) [inline .github/issue_template.md:8]: cannot "report" on the forum so reword that
also point to https://community.openhab.org/t/how-to-ask-a-good-question-help-us-help-you/58396

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+or to this GitHub issue tracker.
```
Comment (mstormi) [inline .github/issue_template.md:9]: give pointer to remove "is: open" from Github standard search as that'll only give open issues in  return so similar or matching closed issues don't show up.
Also prominently add that hint to debug guide.

File: docs/openhabian-DEBUG.md [hunk 3]
```diff
@@ -6,88 +6,204 @@ source: https://github.com/openhab/openhabian/blob/master/docs/openhabian-DEBUG.
...
+However, that does not make your box a "supported" one as we don't have it
+available for our further development and testing. So there remains a risk that
+future openHABian releases will fail to work on your SBC because we changed a
+thing that broke support for your HW - unintentionally however inevitably.
+
+openHABian requires you to provide direct Internet access. Using private IP
+addresses is fine as long as your router properly provides NAT
+(Network Address Translation) services.
+Note: we assume an Ethernet connection to be present at time of installation.
+We do not support installing via WiFi.
```
Comment (mstormi) [inline docs/openhabian-DEBUG.md:38]: I think that's wrong, we *do* support wifi-only. Didn't know by the time I wrote it.
Not sure how to consistently write that down.
Maybe add a sort of flowchart with a section "generic IP connectivity" at the start, then inside that section tell users to *try* with Ethernet when they only tried Wifi ?

File: docs/openhabian-DEBUG.md [hunk 4]
```diff
@@ -6,88 +6,204 @@ source: https://github.com/openhab/openhabian/blob/master/docs/openhabian-DEBUG.
...
-Once you opened the issue, copy `/boot/first-boot.log` from your openHABian box over to your desktop and upload it to GitHub.
-If you succeed logging on and get to see a banner with system information, please also copy that as part of your issue.
-
-If you're able to help in producing a fix to problems, we happily take any Pull Request.
-Explaining git and Github unfortunately is out of our scope (Google is your friend). See also [CONTRIBUTING.md](https://github.com/openhab/openhabian/blob/master/CONTRIBUTING.md).
-For simple fixes to a single file only, you can click through the source starting at <https://github.com/openhab/openhabian> and edit the file online, GitHub will then offer to create the PR.
-You can also clone the openHABian repository, make your changes locally and use git to check in your changes and upload them to a repo copy of yours, then follow the git-offered link to create the PR.
+Search the issues listed there first if 'your' problem has already been seen and
```
Comment (mstormi) [inline docs/openhabian-DEBUG.md:151]: search closed issues, too

File: docs/openhabian-DEBUG.md [hunk 5]
```diff
@@ -6,88 +6,204 @@ source: https://github.com/openhab/openhabian/blob/master/docs/openhabian-DEBUG.
...
+friend).
+See the guidelines outlined in [CONTRIBUTING.md](https://github.com/openhab/openhabian/blob/master/CONTRIBUTING.md)
+as well.
+For simple fixes to a single file only, you can click through the source
+starting at <https://github.com/openhab/openhabian> and edit the file online,
+GitHub will then offer to create the PR.
+You can also clone the openHABian repository, make your changes locally and use
+git to check in your changes and upload them to a repo copy of yours, then
+follow the git-offered link to create the PR.
 Either way, don't forget to sign your work.
```
Comment (mstormi) [inline docs/openhabian-DEBUG.md:178]: Omit signing sentence

File: .github/issue_template.md (hunk 1 above)
Comment (ecdye) [reply .github/issue_template.md]: Better?

File: docs/openhabian-DEBUG.md (hunk 3 above)
Comment (ecdye) [reply docs/openhabian-DEBUG.md]: Dunno how to make a flow chart in MarkDown but is that better?

File: docs/openhabian-DEBUG.md (hunk 4 above)
Comment (ecdye) [reply docs/openhabian-DEBUG.md:148]: Done

File: docs/openhabian-DEBUG.md (hunk 5 above)
Comment (ecdye) [reply docs/openhabian-DEBUG.md]: Done

File: .github/issue_template.md (hunk 1 above)
Comment (mstormi) [reply .github/issue_template.md]: yes

File: docs/openhabian-DEBUG.md (hunk 3 above)
Comment (mstormi) [reply docs/openhabian-DEBUG.md]: me neither. Yes that's ok.

File: .github/issue_template.md (hunk 2 above)
Comment (mstormi) [reply .github/issue_template.md]: ok

**Pull request merged**